js/components/about/about.js

```diff
@@ -0,0 +1,3 @@
+import aboutContent from "../../../README.md"
+
+document.querySelector("#aboutDiv").innerHTML = aboutContent
```

Comment:
Two comments:

1. It's more idiomatic to use `document.getElementById` here
2. Move this code into a [`DOMContentLoaded` event callback](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event)